From 8f6f87ab7de8944060fde0c68f2542c61ce859b5 Mon Sep 17 00:00:00 2001
From: Antony Pavlov <antonynpavlov@gmail.com>
Date: Thu, 6 Jun 2019 22:40:45 +0300
Subject: [PATCH 2/3] litex_liteeth: fix printk %p (ptrval) result

simple '%p' format leads to this:

  liteeth f0003800.mac eth0: irq 0, mapped at (ptrval)

using '%px' format show hex address:

  liteeth f0003800.mac eth0: irq 0, mapped at a0000800

Signed-off-by: Antony Pavlov <antonynpavlov@gmail.com>
---
 drivers/net/ethernet/litex/litex_liteeth.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/litex/litex_liteeth.c b/drivers/net/ethernet/litex/litex_liteeth.c
index df18f3a5f90d..4fac11f5f7ca 100644
--- a/drivers/net/ethernet/litex/litex_liteeth.c
+++ b/drivers/net/ethernet/litex/litex_liteeth.c
@@ -382,7 +382,7 @@ static int liteeth_probe(struct platform_device *pdev)
 		goto err;
 	}
 
-	netdev_info(netdev, "irq %d, mapped at %p\n", netdev->irq, priv->base);
+	netdev_info(netdev, "irq %d, mapped at %px\n", netdev->irq, priv->base);
 
 	return 0;
 err:
-- 
2.20.1

